source/slang/slang-compiler.cpp

```diff
@@ -2177,6 +2177,103 @@ namespace Slang
         return SLANG_OK;
     }
 
+    static void writeString(Stream& stream, const char* string)
+    {
+        stream.write(string, strlen(string));
+    }
+
+    static void escapeDependencyString(const char* string, StringBuilder& builder)
```

Comment:
Nitpik: For variables that are 'output' we have a convention of prefixing with `out` so `outBuilder` here. If it's in/out we prefix with `io`.

Reply:
Done.